### ISSUE TITLE
Support more import/export formats

### ISSUE THREAD

**Issue by j5v** [j5v/xetune#22]
Which formats should XeTune support?

Some future-gazing questions that might help answer that:
1. Would users benefit from interoperability with other apps, and preserve metadata that the `.scl` format loses?
2. Do we need to store XeTune tunings as a portable file, and is a proprietary format required for that?
3. Do other formats store essential data that we cannot infer otherwise?
4. Do we need to persist as a file, data that adds value to tunings, notes, analysis, or configuration profiles. For example, user-provided annotations, generators of tunings, or filtered views?

My instinct is to await user needs (features) that require file persistence, before reaching for proprietary or compatible formats.

**Comment by tank-trax** [comment]
thanks for this amazing project and tool

my biggest request would be the import of SCL files, KBM import is not as crucial for my needs but I can't speak for others, not even sure if keyboard mapping would even be feasible, however being able to set the root note and its frequency would certainly be something worth considering

**Comment by j5v** [comment]
> my biggest request would be the import of SCL files

Good news: you can already drag-and-drop Scala files into XeTune.

**Comment by j5v** [comment]
> KBM import is not as crucial for my needs but I can't speak for others, not even sure if keyboard mapping would even be feasible, however being able to set the root note and its frequency would certainly be something worth considering

Certainly! The workflow is fragmented for reference frequencies. Discussed a bit more in #23.

**Comment by tank-trax** [comment]
> Good news: you can already drag-and-drop Scala files into XeTune.

this is great news, I think I vaguely remember you mentioning this on Discord, thanks for the response
